include/nonius/param.h++

```diff
@@ -0,0 +1,176 @@
+// Nonius - C++ benchmarking tool
+//
+// Written in 2014 by Martinho Fernandes <martinho.fernandes@gmail.com>
```

Comment:
Ha, I see you copied these from some existing stale ones. Since this is PD this notice is not irrelevant, but if you want to claim credit for this file, you're welcome to do so. Otherwise I'll probably do a sweeping change in the repo to something in the spirit of "by the nonius contributors <nonius@rmf.io>"